notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1173 @@
+{
```

Comment:
<p>Can you please check if the above install and uninstall are really necessary? Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Removed.</p>